src/data/security-quiz.ts

```diff
@@ -1,4 +1,31 @@
 const securityQuiz = [
+  {
+    Question:"What is the primary purpose of a wordlist in ethical hacking ?",
+    Answer : "To automate password cracking attempts",
+    Distractor1: "To store user credentials securley",
```

Comment:
You spelled "securely" wrong.

Reply:
Okay, I will make the required changes. Is there anything else? @a2937

Reply:
I had made the required changes please review it.